lib/screens/app/guardians/guardian_user_tile.dart

```diff
@@ -41,16 +41,32 @@ Widget guardianUserTile({MemberModel user, GuardianStatus status, String current
         ),
         tag: "account#${user.account}",
       ),
-      onTap: tileOnTap);
+      onTap: () {
+        tileOnTap(user, status);
+      });
 }
 
 Widget trailingWidget(GuardianStatus status, MemberModel user, String currentUserId) {
   switch (status) {
     case GuardianStatus.requestedMe:
       return Wrap(
         children: [
-          TextButton(child: Text("Accept", style: TextStyle(color: Colors.blue)), onPressed: () {}),
-          TextButton(child: Text("Decline", style: TextStyle(color: Colors.red)), onPressed: () {})
+          TextButton(
+              child: Text("Accept", style: TextStyle(color: Colors.blue)),
+              onPressed: () {
+                FirebaseDatabaseService().acceptGuardianRequestedMe(
+                  currentUserId: currentUserId,
+                  friendId: user.account,
+                );
+              }),
+          TextButton(
+              child: Text("Decline", style: TextStyle(color: Colors.red)),
```

Comment:
needs i18n

Reply:
Ill do all translations at the end. I want to make sure the strings are final before i translate them. 
Once the feature is completed. ill translate them all